beacon_node/beacon_chain/src/validator_monitor.rs

```diff
@@ -487,8 +500,66 @@ impl<T: EthSpec> ValidatorMonitor<T> {
                         u64_to_i64(validator.withdrawable_epoch),
                     );
                 }
+
+                // Add non finalized missed blocks for the monitored validators for the current slot of the current epoch
+                let current_slot = state.slot();
```

Comment:
With the lag implemented this won't catch any missed blocks by checking at `state.slot()`. I think instead we could just separately iterate `missed_blocks` and throw them all into the metric. The metric type could also be changed to an `int_gauge_vec` so that it doesn't increment multiple times for the same missed block.

see: https://prometheus.io/docs/concepts/metric_types/